src/dataServices/dynamoDbParamBuilder.ts

```diff
@@ -207,4 +207,20 @@ export default class DynamoDbParamBuilder {
 
         return params;
     }
+
+    static buildGetActiveSubscriptions(tenantId?: string) {
+        let keyCondition = '_subscriptionStatus = :active';
+        if (tenantId) {
+            keyCondition += ` AND id beginsWith ${tenantId}`;
```

Comment:
This is unsafe. `${tenantId}` may break the DDB syntax. You can instead do `AND id beginsWith :tenantId` and add the tenant id to `ExpressionAttributeValues`.

In general it is the best practice to always use `ExpressionAttributeValues`

Reply:
Tested against DDB and revised the syntax